haero/atmosphere.hpp

```diff
@@ -51,9 +51,15 @@ class Atmosphere final {
   /// liquid water mass mixing ratio [kg vapor/kg dry air]
   ColumnView liquid_mixing_ratio;
 
+  /// grid box averaged cloud liquid number [#/kg dry air]
```

Comment:
is it worth calling this a number mixing ratio?

Reply:
(In the comment, I mean--the variable name is fine)

Reply:
I actually went back and changed it in the variable name as well, it is long but useful.